### DISCUSSION TITLE
Add price feed for XRP/USD to Razor Network

### DISCUSSION THREAD

**Discussion by yohanelly95** [razor-network/governance#18] [General]
## Objective

Proposal for a new price feed for **XRP/USD**

### Description

Datasources Pull Request: https://github.com/razor-network/datasources/pull/72

**Comment by oblo78** [comment]
Ripple is one of the oldest crypto projects. Their community is great and their token, xrp, has a very high market cap. For these reasons i propose to add a new pice feed (XRP/USD) to Razor Network.

**Reply by yohanelly95** [reply]
I agree, was there any specific project that this XRp/USD price feed would be used for? @oblo78

**Reply by oblo78** [reply]
I think it can be useful for example to prediction market projects like Polkamarkets (via Moonbeam integration), SX Network, etc.. @yohanelly95
